packages/sdk/src/types/runtime/actor.ts

```diff
@@ -657,4 +668,32 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
                 .catch(reason => log.error('app', reason));
         }
     }
+
+    private _generateColliderParams(

```

Comment:
>_generateColliderParams [](start = 12, length = 23)

The leading underscore should cause a lint failure. It's fine to disable it for this line. I'm eventually going to update the tslint rule to allow underscore on private members.

Reply:
There is no linter failure that happened for me.  I though we were using this for private function names.  Either way, no error for me.

Reply:
OK. It is fine to use them on private functions if you want to. The lint rule needs some adjustment.

---
In reply to: [257434859](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/203#discussion_r257434859) [](ancestors = 257434859)